packages/ui-components/src/components/UIDialog/UIDialog.tsx

```diff
@@ -206,6 +206,14 @@ export class UIDialog extends React.Component<DialogProps, DialogState> {
      */
     render(): JSX.Element {
         const {
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            acceptButtonText,
```

Comment:
Why is unused variable needed here?

Reply:
@devinea 

Full code is:
```
const {
            // eslint-disable-next-line @typescript-eslint/no-unused-vars
            acceptButtonText,
            cancelButtonText,
            onAccept,
            onCancel,
            acceptButtonId,
            cancelButtonId,
            footer,
            closeButtonVisible,
            multiLineTitle,
            scrollArea = UIDialogScrollArea.Content,
            children,
            ...rest
        } = this.props;
```

then `rest` is used to deepMerge properties. In summary by adding unused variable we exclude it from `rest`

Reply:
I extended the [eslintrule](https://github.com/SAP/open-ux-tools/pull/692/files#diff-c42f01ebf3490ee1be748e94d235263e6c5c0d7dd09ba3429788f5e214c519a8R23) to `ignoreRestSiblings` so i removed the inline comment to ignore `unusedvars`.

Reply:
Thanks for info and update.